docs/book/src/reference/project-config.md

```diff
@@ -2,7 +2,74 @@
 
 ## Overview
 
-The Project Config represents the configuration of a Kubebuilder project. All projects that are scaffolded with the CLI will generate the `PROJECT` file in the projects' root directory.
+The Project Config represents the configuration of a KubeBuilder project. All projects that are scaffolded with the CLI will generate the `PROJECT` file in the projects' root directory. Therefore, it will store all plugins and input data used to generate the project and APIs to better enable plugins to make useful decisions when scaffolding.
+
+## Example
+
+Following an example of an PROJECT file config file which is the result of an project generated with two APIs using the [Deploy Image Plugin][deploy-image-plugin].
+
+```yaml
+# Code generated by tool. DO NOT EDIT.
+# This file is used to track the info used to scaffold your project
+# and allow the plugins properly work.
+# More info: https://book.kubebuilder.io/reference/project-config.html
+domain: testproject.org
+layout:
+- go.kubebuilder.io/v4
+plugins:
+  deploy-image.go.kubebuilder.io/v1-alpha:
+    resources:
+    - domain: testproject.org
+      group: example.com
+      kind: Memcached
+      options:
+        containerCommand: memcached,-m=64,-o,modern,-v
+        containerPort: "11211"
+        image: memcached:1.4.36-alpine
+        runAsUser: "1001"
+      version: v1alpha1
+    - domain: testproject.org
+      group: example.com
+      kind: Busybox
+      options:
+        image: busybox:1.28
+      version: v1alpha1
+projectName: project-v4-with-deploy-image
+repo: sigs.k8s.io/kubebuilder/testdata/project-v4-with-deploy-image
+resources:
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: testproject.org
+  group: example.com
+  kind: Memcached
+  path: sigs.k8s.io/kubebuilder/testdata/project-v4-with-deploy-image/api/v1alpha1
+  version: v1alpha1
+  webhooks:
+    validation: true
+    webhookVersion: v1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: testproject.org
+  group: example.com
+  kind: Busybox
+  path: sigs.k8s.io/kubebuilder/testdata/project-v4-with-deploy-image/api/v1alpha1
+  version: v1alpha1
+version: "3"
+
+## Why do we need to store the plugins and data used?
```

Comment:
Seems like we need to close the code block above by ` ``` ` ?

Because the sample layout seems incorrect:
https://deploy-preview-3287--kubebuilder.netlify.app/reference/project-config.html?highlight=project#project-config

Reply:
Sure done that